brands/amenity/fuel.json

```diff
@@ -1,4 +1,20 @@
 {
+  "amenity/fuel|中国石油": {
+    "countryCodes": ["cn"],
+    "matchNames": ["中油加油站", "CNPC", "uSmile", "中国石油天然气集团公司", "CNPC fuel"],
+    "tags": {
+      "amenity": "fuel",
+      "brand": "中国石油",
+      "brand:en": "PetroChina",
+      "brand:wikidata": "Q503182",
+      "brand:wikipedia": "en:PetroChina",
+      "brand:zh": "中国石油",
+      "name": "中国石油",
+      "name:en": "PetroChina",
+      "name:zh": "中国石油",
+      "operator": "中国石油天然气集团公司"
```

Comment:
Thanks!  Can you remove this `operator` tag?  We don't currently include `operator` tags in this index. (see #2816 for context)

The other matchnames seem ok.

Reply:
Done ;-)